### PR TITLE
fix remaining trustDomain settings

### PR DESCRIPTION
**Pull request by richardwxn**

### CODE REVIEW

File: asm/istio/options/hub-meshca.yaml [hunk 1]
```diff
@@ -30,16 +30,15 @@ spec:
           - name: TOKEN_AUDIENCES
             value: "istio-ca,ENVIRON_PROJECT_ID.hub.id.goog" # {"$ref":"#/definitions/io.k8s.cli.substitutions.token-audiences-hub"}
   meshConfig:
+    trustDomain: "ENVIRON_PROJECT_ID.hub.id.goog" # {"$ref":"#/definitions/io.k8s.cli.substitutions.trust-domain-hub"}
     defaultConfig:
       proxyMetadata:
         GCP_METADATA: "PROJECT_ID|PROJECT_NUMBER|asm-cluster|us-central1-c" # {"$ref":"#/definitions/io.k8s.cli.substitutions.gke-hub-metadata"}
-        TRUST_DOMAIN: "ENVIRON_PROJECT_ID.hub.id.goog" # {"$ref":"#/definitions/io.k8s.cli.substitutions.trust-domain-hub"}
         GKE_CLUSTER_URL: "https://gkehub.googleapis.com/projects/ENVIRON_PROJECT_ID/locations/global/memberships/MEMBERSHIP_ID" # {"$ref":"#/definitions/io.k8s.cli.substitutions.hub-idp-url"}
     trustDomainAliases: # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.trustDomainAliases"}
```
Comment (chizhg) [inline asm/istio/options/hub-meshca.yaml:38]: Shall we delete this one as well?

Comment (richardwxn) [reply asm/istio/options/hub-meshca.yaml:38]: you mean the trustDomainAliases? they are different settings

**Comment by richardwxn** [comment]
@zerobfd @elfinhe PTAL

File: asm/istio/options/hub-meshca.yaml (hunk 1 above)
Comment (chizhg) [reply asm/istio/options/hub-meshca.yaml:38]: Got it. Thanks.

**Review by zerobfd — APPROVED** [review]

**Pull request merged**